Jasmine/Home/ViewControllers/HomeScreenViewController.swift

```diff
@@ -55,9 +55,9 @@ class HomeScreenViewController: UIViewController {
             let viewModel = PhrasesExplorerViewModel(phrases: ciHuiGameData.phrases)
 
             if (sender as? UIButton) === phrasesWithSelectButton {
-                phrasesExplorer.segueWith(viewModel, isMarkable: true)
+                phrasesExplorer.segueWith(viewModel, onSaveCallBack: { print($0) })
```

Comment:
This is just the debug screen, for the actual use a function from previous VC will be registered as callback (see designer VC)